templates/zabbix-agent-systemd.init.erb

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=Zabbix Agent
+After=syslog.target network.target
+
+[Service]
+Type=oneshot
```

Comment:
You are right acording to [redhat documentation](https://access.redhat.com/documentation/en-US/Red_Hat_Enterprise_Linux/7/html/System_Administrators_Guide/sect-Managing_Services_with_systemd-Unit_Files.html).

I'll fix it along with your other comments.